.circleci/config.yml

```diff
@@ -180,20 +180,22 @@ jobs:
       - run: pip install --upgrade httplib2 jira oauth2client pylint isort==4.3.8
       - run: pip install -I alembic==0.9.1
       - run: ./ci/setup.sh
+
+      - run: pip install -r requirements.txt # installs google libraries
+      - run: wget https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 -O cloud_sql_proxy # downloads the gcp
+      - run: chmod +x cloud_sql_proxy # makes proxy executable
+      - run: sudo ln -s ~/cloud_sql_proxy /usr/local/bin/cloud_sql_proxy # link to where the alembic script is looking
       - run:
           name: activate credentials
           working_directory: ~/all-of-us/raw-data-repository/
           command: |
-            pyenv local 2.7.12
             ./ci/activate_creds.sh ~/gcloud-credentials.key
-            pyenv local 3.7.0
       - run:
           environment:
-            PYTHONPATH: ~/all-of-us/raw-data-repository/rdr_service/
+            PYTHONPATH: ~/all-of-us/raw-data-repository/
           working_directory: ~/all-of-us/raw-data-repository/rdr_service/
           command:  python -m tools alembic --project https://pmi-drc-api-test.appspot.com upgrade head
```

Comment:
In my testing I also had to change this command to the following to avoid the `invalid project name [https://pmi-drc-api-test.appspot.com]` error.

`python -m tools alembic --project pmi-drc-api-test upgrade head`

Reply:
done